### PR TITLE
#2432 Fix: error when decrypting pgpmime with inline image

### LINKED ISSUE #2432: Error when decrypting PGP/MIME message with inline image
App shows `Error: Misformed armored text` error message when trying to decrypt PGP/MIME with inline image:

Sample email - `PGP/MIME message with inline image for flowcrypt-ios #2432` on `flowcrypt.compatibility` account

### PR DESCRIPTION
**Pull request by ioanmo226**
This PR fixed error when decrypting pgpmime with inline image

close #2432 // if this PR closes an issue

----------------------------------

**Tests** _(delete all except exactly one)_:
- Tests added or updated

--------------------------------

### To be filled by reviewers

I have reviewed that this PR... _(tick whichever items you personally focused on during this review)_:
- [ ] addresses the issue it closes (if any)
- [ ] code is readable and understandable
- [ ] is accompanied with tests, or tests are not needed
- [ ] is free of vulnerabilities
- [ ] is documented clearly and usefully, or doesn't need documentation

### CODE REVIEW

**Review by sosnovsky — COMMENTED** [review]
Message is decrypted without errors, but I can't open attachment from `PGP/MIME message with inline image for flowcrypt-ios #2432` thread.
It's an image, so attachment preview should work, but app only proposes to download attachment. And even after download it can't be opened, because it's downloaded as encrypted file (ending at `.jpg.pgp`).

Can you please check this issue? If it's a small fix then we can include it in the current PR, otherwise I'll work on it in a separate issue.

File: FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift [hunk 1]
```diff
@@ -137,7 +137,7 @@ final class ThreadDetailsViewController: TableNodeViewController {
 
                 var processedMessage = try await getAndProcessMessage(
                     identifier: rawMessage.identifier,
-                    onlyLocalKeys: true,
+                    onlyLocalKeys: false,
```
[the remark this answers is not in the document]
Comment (sosnovsky) [reply FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift:140]: Hmm, I think we use only local keys here to make decryption process faster (as we don't need to wait for keys fetch request), and if there are no local keys found - then they should be fetched after decryption process is completed. As I remember, it should work this way.
Did you have any issues when opening message from sender which is not in your contacts?

[the remark this answers is not in the document]
Comment (ioanmo226) [reply FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift:140]: Ohh. Yep. You are right.
I overlooked it.

[the remark this answers is not in the document]
Comment (ioanmo226) [reply FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift:140]: Thank you.

[the remark this answers is not in the document]
Comment (sosnovsky) [reply FlowCrypt/Controllers/Threads/ThreadDetailsViewController.swift:140]: Let's also add comment here saying `using only local keys for faster decryption. remote keys are fetched after decryption process is finished`, so in the future it'll be more clear why only local keys are used.

**Review by sosnovsky — APPROVED** [review]
Well done 👍

**Pull request merged**